crates/types/src/config_loader.rs

```diff
@@ -22,7 +22,7 @@ use tracing::{error, info, warn};
 #[derive(thiserror::Error, codederror::CodedError, Debug)]
 #[code(restate_errors::RT0002)]
 pub enum ConfigLoadError {
-    #[error("configuration loading error: {0}")]
+    #[error("configuration loading error: {0:?}")]
```

Comment:
Oh, nvm, I'll take this out! This helped me troubleshoot the conflict in the top-level `connection-timeout` property because we flatten both HTTP and the Networking options into the top-level struct.